src/python/pants/backend/codegen/grpcio/grpcio_run.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import functools
+import logging
+from collections import OrderedDict
+
+from pants.backend.codegen.grpcio.grpcio_prep import GrpcioPrep
+from pants.backend.codegen.grpcio.python_grpcio_library import PythonGrpcioLibrary
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.base.build_environment import get_buildroot
+from pants.base.exceptions import TaskError
+from pants.base.workunit import WorkUnitLabel
+from pants.task.simple_codegen_task import SimpleCodegenTask
+from pants.util.contextutil import pushd
+
+
+class GrpcioRun(SimpleCodegenTask):
+  """[XAIN] Task to compile protobuf"""
```

Comment:
We generally don't put (nick)names in files, so please remove this.

Reply:
My bad - it's done now 👍